index.html

```diff
@@ -39,6 +39,178 @@ <h2 id="profession">I'm a software developer</h2>
+                you need coded, don't hesitate to contact me.
+            </p>
+            <button type="button" class="see_button">Get my resume</button>
+        </div>
+
+        <div class="proficiency">
+            <div class="skill_card">
+                <div class="skill-card_icon">
+                    <img src="Assets/Icons/languages icon.png" alt="Languages icon">
+                </div>
+                <h4 class="category">Languages</h4>
+                <ul class="skills_list2">
+                    <li class="capabilities">Javascript</li>
+                    <li class="capabilities">Ruby</li>
+                    <li class="capabilities">HTML</li>
+                    <li class="capabilities">CSS</li>
+                </ul>
+            </div>
+
+            <div class="skill_card">
+                <div class="skill-card_icon">
+                    <img src="Assets/Icons/Frameworks icon.png" alt="Frameworks icon">
+                </div>
+                <h4 class="category">Frameworks</h4>
+                <ul class="skills_list2">
+                    <li class="capabilities">React.js</li>
+                    <li class="capabilities">Ruby on Rails</li>
+                    <li class="capabilities">RSpec</li>
+                    <li class="capabilities">Capybara</li>
+                    <li class="capabilities">Selenium</li>
+                </ul>
+            </div>
+
+            <article class="skill_card">
+                <div class="skill-card_icon">
+                    <img src="Assets/Icons/skills icon.png" alt="Skills icon">
+                </div>
+                <h4 class="category">Skills</h4>
+                <ul class="skills_list2">
+                    <li class="capabilities">Database Management</li>
+                    <li class="capabilities">Version Control</li>
+                    <li class="capabilities">CLI</li>
+                    <li class="capabilities">Web Development</li>
+                    <li class="capabilities">API Design</li>
+                </ul>
+            </div>
+        </div>
+    </section>
+
+    <section class="contact_me" id="contact">
+        <div class="parting_shot">
+            <p>
+                I'm always interested in hearing about new projects, 
+                so if you'd like to chat, please get in touch.
+            </p>
+        </div>
+
+        <form action="https://formspree.io/f/xwkjdnzq" method="post" class="contact_form" id="hello">
+            <input type="text" name="Full name" id="name" placeholder="Full name" required>
+            <input type="text" name="email" id="email" placeholder="Email address" required>
+            <textarea name="Get in touch" id="message" placeholder="Hey, I'd like to connect to..." cols="30" rows="500" maxlength="500"></textarea>
+            <button type="submit" class="say_hello">Get in touch</button>
+            <span class="email_display">lhinga75@gmail.com</span>
+        </form>
```

Comment:
Good job implementing the form requirements for this project. 🤝 However there are some changes I would recommend you make. ♻️ 

- [ ] Kindly ensure that your `name` input has a `max-length` of `30` as the project requirements suggest.

- [ ] Please ensure that your `textarea` input has a `required` property as the project requirement suggest. 📜 Currently your form can submit without a `message` input.

Reply:
Thank you I will get to it.